lstchain/visualization/plot_dl2.py

```diff
@@ -696,3 +699,60 @@ def direction_results(dl2_data, points_outfile=None, plot_outfile=None):
         fig.savefig(plot_outfile)
 
     return fig, axes
+
+
+def plot_wobble(source_position, n_points, ax = None):
+    """
+    Plot 2D map of ON/OFF positions w.r.t. to the camera center
```

Comment:
incomplete docstring, it would be great if you could add info on inputs/outputs please